lib/games_manager.rb

```diff
@@ -32,14 +32,21 @@ def highest_total_score
     end.total_game_score
   end
 
+  def percentage_ties
+    ties = @games.count do |game|
+      !game.home_is_winner? && !game.visitor_is_winner?
```

Comment:
The game #winner_id method will return 'tie' for games that are a tie.

Reply:
I'm inclined to agree, but winner_id already calls both other methods, so I don't see a difference from an efficiency standpoint.